src/app/domain/case-viewer/components/summary-panel/summary-panel.component.html

```diff
@@ -1,7 +1,14 @@
 <div data-selector="summary-panel-component" class="govuk-grid-row">
-    <ng-container *ngFor="let section of panelData.sections">
-        <div class="govuk-grid-column-one-half">
-            <app-data-list [title]="section.name" [dataList]="section.fields"></app-data-list>
-        </div>
-    </ng-container>
+    <div class="govuk-grid-column-two-thirds">
+        <ng-container *ngFor="let section of panelData.sections">
+            <div  class="govuk-grid-row">
+                <div class="govuk-grid-column" *ngIf="section.id !== 'recentEvents'">
+                    <app-data-list [title]="section.name" [dataList]="section.fields"></app-data-list>
+                </div>
+            </div>
+        </ng-container>
+    </div>
+    <div class="govuk-grid-column-one-third">
+        <app-timeline-section [events]="getRecentEvents()" [caseId]="caseId"></app-timeline-section>
```

Comment:
This shouldn't be a function call to an expensive function. This gets called every time Angular runs the digest cycle looking for changes. It's best to assign to a field and use that directly.

Reply:
We usually use ngOnChange then set a property as @Louisblack says